api/src/Serializer/Normalizer/UriTemplateNormalizer.php

```diff
@@ -52,7 +52,7 @@ public function normalize($object, $format = null, array $context = []): array|s
         $result['_links']['oauthGoogle'] = ['href' => $this->urlGenerator->generate('connect_google_start').'{?callback}', 'templated' => true];
         $result['_links']['oauthPbsmidata'] = ['href' => $this->urlGenerator->generate('connect_pbsmidata_start').'{?callback}', 'templated' => true];
         $result['_links']['oauthCevidb'] = ['href' => $this->urlGenerator->generate('connect_cevidb_start').'{?callback}', 'templated' => true];
-        $result['_links']['resetPassword'] = ['href' => $this->urlGenerator->generate('api_reset_passwords_post_collection').'{/id}', 'templated' => true];
+        $result['_links']['resetPassword'] = ['href' => $this->urlGenerator->generate('_api_/reset_password{._format}_post').'{/id}', 'templated' => true];
```

Comment:
It looks like a URL, but it actually is the operation name 😄 

This is new naming scheme from api-platform => _api_ + uriTemplate + method
(and after [this issue](https://github.com/api-platform/core/issues/5203) is resolved also routePrefix)

Reply:
Yes, correct. `connect_cevidb_start` is coming from a custom controller and the route name is directly specified via attribute:
https://github.com/ecamp/ecamp3/blob/devel/api/src/Controller/CevidbController.php#L14

For API resources, api platform will generate a route name for each operation (which also serves as the operation name). This could be overwritten with the `name` property of any `Operation` attribute, but I guess we're ok with the automatic generated ones.